benchmarker/__main__.py

```diff
@@ -59,13 +59,15 @@ def main():
         else:
             result["device"] = result["platform"]["cpu"]["brand"]
         if args.flops:
-            result["problem"]["gflop_measured"] = get_counters(command)
+            result["gflop_measured"] = get_counters(command)
         elif args.fapp_power:
             total, details = fapp.get_power_total_and_detail(command)
             result["problem"]["power"] = {"total": total, "details": details}
     cute_device = get_cute_device_str(result["device"]).replace(" ", "_")
     result["path_out"] = os.path.join(result["path_out"], result["problem"]["name"])
     result["path_out"] = os.path.join(result["path_out"], cute_device)
+    if "gflop_measured" in result.keys():
+        result["gflop_per_joule"] = result["gflop_measured"] / result["time_total"]
```

Comment:
that seems more like flops per second, not really per joule

Reply:
And that's the reason why we do reviews! (and the gifs of course). Any suggestion on where to put it?! Is it fine where it is?!

Reply:
Done! Is that better? `"gflop_per_joule": 1.043047889268956,` with
```
python -m benchmarker --problem=resnet50 --framework=pytorch --problem_size=4 --batch_size=2 --nb_epoch=1 --flops
```